draft-ietf-rtcweb-jsep.xml

```diff
@@ -3174,17 +3164,63 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           processing MUST stop and an error MUST be returned.</t>
 
           <t>Next, the SessionDescription is parsed into a data
-          structure, as described in the
-          <xref target="sec.parsing-a-desc" /> section below. If parsing
+          structure, as described in
+          <xref target="sec.parsing-a-desc" /> below. If parsing
           fails for any reason, processing MUST stop and an error MUST
           be returned.</t>
 
           <t>Finally, the parsed SessionDescription is applied as
-          described in the
-          <xref target="sec.applying-a-remote-desc" /> section
+          described in
+          <xref target="sec.applying-a-remote-desc" />
           below.</t>
         </list></t>
       </section>
+
+      <section title="Processing a Rollback"
+               anchor="sec.processing-a-rollback">
+
+        <t>  The rollback is used to recover from possible errors and
+        return to stable state the system was in before a new offer /
+        answer negotiation started. </t>
+
+        <t> When either setLocalDescription or setRemoteDescription
+        method is called with a a session description of type
+        "rollback" a rollback occurs. </t>
+
+        <t> If the system is currently in the "stable" state, no
```

Comment:
Should this be a no-op or an error? I can see arguments either way; rolling back from a stable state to the same state is a no-op, but if the app was really trying to roll back to the previous state, it should trigger an error.

Generally it seems like this would most likely happen if you were confused about the current state, so an error seems better to me.